accessibility-checker-engine/help/IBMA_Color_Contrast_WCAG2AA_PV.mdx

```diff
@@ -34,7 +34,7 @@ The contrast ratio of text and the background (i.e. background with a color grad
 ## What to do
 
 * Text should always have a solid background (i.e. not a color gradient or a background image) where possible;
-* OR when that is not possible (which is very rare), manually check the contrast ratio of the text against the lightest and the darkest colors of the background with a Color Contrast Analyzer. For large text at least 18 point or at least 14 point when bold, the contrast ratio must be at least 3:1. For smaller text, it should be at least 4.5:1. If less, adjust the colors. 
+* OR when that is not possible (which is very rare), manually check the contrast ratio of the text against the lightest and the darkest colors of the background with a [Color Contrast Analyzer](https://www.ibm.com/able/toolkit/verify/manual/#color). For regular text it must be at least 4.5:1. For large text at least 18 point or at least 14 point when bold, the contrast ratio must be at least 3:1. If less, adjust the colors.
```

Comment:
In the earlier rule, as suggested by our content designer, we used '4.5 to 1'.  For consistency we should do the same here.

For regular text it must be at least 4.5 to 1. For large text at least 18 point or at least 14 point when bold, the contrast ratio must be at least 3 to 1.

Reply:
Done, thanks.